From 0739363c1d54c9adf4cad1b8e72d087e1929582c Mon Sep 17 00:00:00 2001
From: Zenithal <i@zenithal.me>
Date: Sun, 6 Feb 2022 16:01:36 +0800
Subject: [PATCH] No explicit -lgcc

For Clang/LLVM user with NO cross gcc installed,
there is no libgcc available; At this time user
can manually export LIBS=-lclang-rt.

For cross gcc user, -lgcc is usually added by the
compiler thus there is no need to add this flag
in config file.
Only when -nostdlib/-nodefaultlibs is specified
should we add -lgcc, which is not the case here
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 81492de5..8801d8e6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -96,7 +96,7 @@ LDFLAGS="$LDFLAGS -Wl,--build-id=none"
 
 AC_SUBST(CFLAGS)
 AC_SUBST(LDFLAGS)
-AC_SUBST([LIBS], ["-lgcc"])
+AC_SUBST(LIBS)
 AC_SUBST(WITH_ARCH)
 AC_SUBST(host_alias)
 
